libgadget/allvars.c

```diff
@@ -22,7 +22,6 @@
 
 
 int ThisTask;			/*!< the number of the local processor  */
```

Comment:
It looks weird NTask is gone but ThisTask is still here. I guess it still used in too many places?

Reply:
Perhaps add a TODO / FIXME about planning to refactor it away in the future.

Reply:
It is not that many places but I ran out of steam...we shall get rid of it at some point.